### ISSUE TITLE
Flag "write-mask-snplist" not available?

### ISSUE THREAD

**Issue by IoannaTach** [rgcgithub/regenie#188]
Is flag "write-mask-snplist" not available for the latest REGENIE release please?

**Comment by joellembatchou** [comment]
If you run `./regenie --helpFull`, you should see the list of available options (--write-mask-snplist should be one of them).

**Comment by IoannaTach** [comment]
Then why am I getting this error???

ERROR: Option ‘write-mask-snplist’ does not exist

**Comment by joellembatchou** [comment]
I checked it with the released Regenie v2.2.4 binary (https://github.com/rgcgithub/regenie/releases/download/v2.2.4/regenie_v2.2.4.gz_x86_64_Linux_mkl.zip)

```
> wget https://github.com/rgcgithub/regenie/releases/download/v2.2.4/regenie_v2.2.4.gz_x86_64_Linux_mkl.zip
> unzip regenie_v2.2.4.gz_x86_64_Linux_mkl.zip
> ./regenie_v2.2.4.gz_x86_64_Linux_mkl --write-mask-snplist

              |=============================|
              |      REGENIE v2.2.4.gz      |
              |=============================|

Copyright (c) 2020-2021 Joelle Mbatchou, Andrey Ziyatdinov and Jonathan Marchini.
Distributed under the MIT License.
Compiled with Boost Iostream library.
Using Intel MKL with Eigen.

ERROR: You must provide an output prefix using '--out'
For more information, use option '--help' or visit the website: https://rgcgithub.github.io/regenie/
```
As you can see the option is being recognized (otherwise it would have printed `ERROR: Option ‘XXX’ does not exist`)

**Comment by IoannaTach** [comment]
Then can you please identify what I a doing wrong here and getting the error "ERROR: Option ‘write-mask-snplist’ does not exist"? Thank you, much appreciated!

regenie \
  --step 2 \
  --bed /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/Extract_step2/${trait}_${model}_${chrom} \
  --covarFile /projects/cgr/users/kvrf354/extract_pedmap_step1/Covariates_EURandUnrelated_PCs_TotalSyn.txt \
  --phenoFile /projects/cgr/users/kvrf354/extract_pedmap_step1/phenotypes_final.txt \
  --phenoColList allcomers,AKI,IGAN,FSGS,GN,CYS,TIN,HT,DKD \
  --covarColList Sex,Total.Sample.Variant.Synonymous,PC_EURandUnrelated_1,PC_EURandUnrelated_2,PC_EURandUnrelated_3,PC_EURandUnrelated_4,PC_EURandUnrelated_5,PC_EURandUnrelated_6,PC_EURandUnrelated_7,PC_EURandUnrelated_8,PC_EURandUnrelated_9,PC_EURandUnrelated_10,PC_EURandUnrelated_11,PC_EURandUnrelated_12,PC_EURandUnrelated_13,PC_EURandUnrelated_14,PC_EURandUnrelated_15 \
  --catCovarList Sex \
  --bt \
  --firth 0.01 --approx \
  --test dominant \
  --pred /projects/cgr/users/kvrf354/extract_pedmap_step1/Portfolio/REGENIE_step1/fit_bin_out_portfolio_dominant_pred.list \
  --bsize 200 \
  --aaf-bins ${max_maf} \
  --anno-file /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/${trait}_${chrom}_${model}_annotation.txt \
  --set-list /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/${trait}_${chrom}_${model}_genelist.txt \
  --mask-def /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/${model}_mask.txt \
  --write-mask /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/REGENIE/${trait}/${trait}_${model}_${chrom}_dominant_mask \
  --write-mask-snplist /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/REGENIE/${trait}/${trait}_${model}_${chrom}_dominant_mask_snplist \
  --out /projects/cgr/users/kvrf354/Portfolio_REGENIE_genebased/REGENIE/${trait}/${trait}_${model}_${chrom}_dominant_burden

**Comment by joellembatchou** [comment]
Can you include the full output of  running `regenie --helpFull`?

I would recommend using [one of the pre-compiled binaries](https://github.com/rgcgithub/regenie/releases/tag/v2.2.4) of the latest release.

**Comment by IoannaTach** [comment]
regenie --write-mask-snplist
              |=============================|
              |      REGENIE v2.0.2.gz      |
              |=============================|

Copyright (c) 2020 Joelle Mbatchou and Jonathan Marchini.
Distributed under the MIT License.
Compiled with Boost Iostream library.
Using Intel MKL with Eigen.

ERROR: Option ‘write-mask-snplist’ does not exist
For list of arguments, run with option --help

**Comment by IoannaTach** [comment]
And it is not listed under regenie --helpFull

**Comment by joellembatchou** [comment]
Try the Centos version (which does not require that library)

**Comment by joellembatchou** [comment]
Also note that the option was not available in v2.0.2 which is why you are getting the error.
